index.src.html

```
@@ -643,6 +643,22 @@ <h3 id='preloader'>
 			or one which it cannot correctly determine the truth value of,
 			it must not attempt to preload <strong>any</strong> URL for the given <a element>img</a> element.
 	</ul>
+	<div class='note'>
+		In some cases, the preload scanner or the parser cannot be aware of the final viewport's dimensions when it encounters a <a element>picture</a> or a
+		<a element>img</a> element. One example of such a case is when a '@viewport' definition is located in an external style-sheet. Another
+		is when the preload scanner is operating on a <a element>iframe</a> based document, which has its dimensions defined in an external
+		style-sheet.
+		<p>
+		Authors should be aware of these limitations, and should follow the following best practices:
+		<ul>
+			<li>
+				''@viewport'' CSS rules should be defined inline whenever viewport dependant resources need to be fetched as part of the
+				page.
+			<li>
+				iframes that may contain viewport dependant resources should have their dimensions defined either in markup (as <a
+					attribute>height</a> and <a attribute>weight</a> attributes) or as inline CSS rules.
```

Personal style I guess, but please don't break in the middle of a tag?  Preferably do semantic linebreaking, where you break lines between signficant phrases.  It tends to keep your lines at 80-100 chars anyway.  At minimum, though, just push the start of the tag down to the next line.

Also, "attribute" is a WebIDL definition type.  You want "element-attr". Preferably specify the element as well, as `<a element-attr for=iframe>width</a>`.  (You'll have to silence the error in the "Ignored Terms" metadata, because these things are not adequately marked up in HTML for Bikeshed to link to them, but writing them correctly now will auto-upgrade them when the targets _are_ marked up correctly.)